src/components/Content/index.tsx

```diff
@@ -94,7 +97,13 @@ export const Content: React.VFC = () => {
 
         <Route component={PageDoesNotExist} />
       </Switch>
-      {hasUsedSpruceBefore && <WelcomeModal />}
+      {hasUsedSpruceBefore && (
```

Comment:
I am possibly doing something wrong but it seems to think I'm a new user every time now :< 

(I'm seeing this when testing on `prod`, `staging` right now doesn't have your backend changes I believe)

https://user-images.githubusercontent.com/47064971/167502704-f2c8bd32-b928-4a21-a936-30e47f9f8ea8.mov

Reply:
nvm it seems like you just fixed it :]